website/docusaurus.config.js

```diff
@@ -51,20 +48,20 @@ module.exports={
   "plugins": [],
   "themeConfig": {
     "prism": {
-      "additionalLanguages": ["typescript","javascript","json","csharp"],
-      "theme": require('prism-react-renderer/themes/vsDark')
+      "additionalLanguages": ["typescript", "javascript", "json", "csharp"],
+      "theme": require('prism-react-renderer').themes.vsDark
```

Comment:
I'm getting build errors on this line because `prism-react-renderer` is not included as an explicit dependency in this package.

Reply:
oddly I don't get errors there (tested on a clean checkout) - but it doesn't hurt to resolve this regardless. Done!